help/en/html/tools/logixng/reference/chapter11_1.shtml

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="generator" content="HTML Tidy for HTML5 for Apple macOS version 5.8.0">
+  <title>LogixNG Reference - Chapter 11.1</title>
+  <meta name="author" content="Daniel Bergqvist">
+  <meta name="author" content="Dave Sand">
+  <meta name="keywords" content="jmri LogixNG reference table icon">
+  <!--#include virtual="/help/en/parts/Style.shtml" -->
+</head>
+<body>
+  <!--#include virtual="/help/en/parts/Header.shtml" -->
+
+  <div id="mBody">
+    <div id="mainContent" class="no-sidebar">
+      <h1>LogixNG Reference - Chapter 11.1</h1>
+
+      <h2>LogixNG Table Icon</h2>
+
+      <p>The <strong>LogixNG table</strong> icon can be added to a <strong>Panel Editor</strong>
+      panel to display the contents of a LogixNG table.  The table can be read only, editable or
+      only selected columns are editable. It is also possible to have columns with <em>buttons</em>
+      or <em>dropdown lists</em>.</p>
+
+      <p>The <strong>LogixNG panel</strong> icon type is selected from the <strong>Panel Editor</strong>
+      icon type list.  This will display a list LogixNG tables.</p>
```

Comment:
It seems to the that the word "of" is missing between "list" and "LogixNG":
`This will display a list of LogixNG tables.`